src/main/java/com/teamacronymcoders/epos/client/gui/QuiverGui.java

```diff
@@ -0,0 +1,131 @@
+package com.teamacronymcoders.epos.client.gui;
+
+import com.hrznstudio.titanium.api.client.AssetTypes;
+import com.hrznstudio.titanium.api.client.IAsset;
+import com.hrznstudio.titanium.api.client.IGuiAddon;
+import com.hrznstudio.titanium.client.gui.IGuiAddonConsumer;
+import com.hrznstudio.titanium.client.gui.addon.ICanMouseDrag;
+import com.hrznstudio.titanium.client.gui.addon.IClickable;
+import com.hrznstudio.titanium.client.gui.addon.SlotsGuiAddon;
+import com.hrznstudio.titanium.client.gui.asset.IAssetProvider;
+import com.mojang.blaze3d.platform.GlStateManager;
+import com.teamacronymcoders.epos.container.QuiverContainer;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.IHasContainer;
+import net.minecraft.client.gui.screen.inventory.ContainerScreen;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.entity.player.PlayerInventory;
+import net.minecraft.inventory.container.Container;
+import net.minecraft.inventory.container.INamedContainerProvider;
+import net.minecraft.util.text.ITextComponent;
+import net.minecraft.util.text.TextFormatting;
+import net.minecraft.util.text.TranslationTextComponent;
+import org.lwjgl.glfw.GLFW;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.List;
+
+public class QuiverGui extends ContainerScreen<QuiverContainer> implements IGuiAddonConsumer, IHasContainer<QuiverContainer>, INamedContainerProvider {
+    private final QuiverContainer container;
+    private IAssetProvider assetProvider;
+    private int x;
```

Comment:
what is x? what is y?

Reply:
x & y are set in drawGuiContainerBackgroundLayer which is equal to:
x = (width - xSize) / 2;
y = (height - ySize) / 2;

Reply:
Okay sooooo it's the center? Need better naming.